xdp-filter: GEN xdp-filter.8: `Debugger entered--Lisp error: (file-missing "Searching for program" "No such file or directory" "/usr/bin/env bash")`

Trying to build xdp-tools 1.2.8 in OpenWrt fails on my system:

```
$ git log --oneline --no-decorate -1
aa12a0fdd1 dnsmasq: add option to expose additional paths to jail
$ make -j1 V=sc
[…]
make[3]: Entering directory '/scratch/local2/ow/package/network/utils/xdp-tools'
rm -f /scratch/local2/ow/build_dir/target-aarch64_cortex-a53_musl/xdp-tools-1.2.8/.built
touch /scratch/local2/ow/build_dir/target-aarch64_cortex-a53_musl/xdp-tools-1.2.8/.built_check
CFLAGS="-Os -pipe -mcpu=cortex-a53 -fno-caller-saves -fno-plt -fhonour-copts -Wno-error=unused-but-set-variable -Wno-error=unused-result -ffile-prefix -map=/scratch/local2/ow/build_dir/target-aarch64_cortex-a53_musl/xdp-tools-1.2.8=xdp-tools-1.2.8 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro    -I/scratch/local2/ow/staging_dir/toolchain-aarch64_cortex-a53_gcc-11.3.0_musl/usr/include -I/scratch/local2/ow/staging_dir/toolchain-aarch64_cortex-a53_gcc-11.3.0_musl/include/fortify -I/scratch/local2/ow/staging_dir/toolchain-aarch64_cortex-a53_gcc-11.3.0_musl/include   " CXXFLAGS="-Os -pipe -mcpu=cortex-a53 -fno-caller-saves -fno-plt -fhonour-copts -Wno-error=unused-but-set-variable -Wno-error=unused -result -ffile-prefix-map=/scratch/local2/ow/build_dir/target-aarch64_cortex-a53_musl/xdp-tools-1.2.8=xdp-tools-1.2.8 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro    -I/scratch/local2/ow/staging_dir/toolchain-aarch64_cortex-a53_gcc-11.3.0_musl/usr/include -I/scratch/local2/ow/staging_dir/toolchain-aarch64_cortex-a53_gcc-11.3.0_musl/include/fortify -I/scratch/local2/ow/staging_dir/toolchain-aarch64_cortex-a53_gcc-11.3.0_musl/include   " LDFLAGS="-L/scratch/local2/ow/staging_dir/toolchain-aarch64_cortex-a53_gcc-11.3.0_musl/usr/lib -L/scratch/local2/ow/staging_dir/toolchain-aarch64_cortex-a53_gcc-11.3.0_musl/lib -znow -zrelro    " PREFIX=/usr RUNDIR=/tmp/run make  -C /scratch/local2/ow/build_dir/target-aarch64_cortex-a53_musl/xdp-tools-1.2.8/. AR="aarch64-openwrt-linux-musl-gcc-ar" AS="aarch64-openwrt-linux-musl-gcc -c -Os -pipe -mcpu=cortex -a53 -fno-caller-saves -fno-plt -fhonour-copts -Wno-error=unused-but-set-variable -Wno-error=unused-result -ffile-prefix-map=/scratch/local2/ow/build_dir/target-aarch64_cortex-a53_musl/xdp-tools-1.2.8=xdp-tools-1.2.8 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro  " LD=aarch64-openwrt-linux-musl-ld NM="aarch64-openwrt-linux-musl-gcc-nm" CC="aarch64-openwrt-linux-musl-gcc" GCC="aarch64-openwrt-linux-musl-gcc" CXX="aarch64-openwrt-linux-musl-g++" RANLIB="aarch64-openwrt-linux-musl-gcc-ranlib" STRIP=aarch64-openwrt-linux-musl-strip OBJCOPY=aarch64-openwrt-linux-musl-objcopy OBJDUMP=aarch64-openwrt-linux-musl-objdump SIZE=aarch64-openwrt-linux-musl-size CROSS="aarch64-openwrt-linux-musl-" ARCH="aarch64" ;
make[4]: Entering directory '/scratch/local2/ow/build_dir/target-aarch64_cortex-a53_musl/xdp-tools-1.2.8'

lib

  libxdp
                                                                                                                                                      
  util
make[6]: Nothing to be done for 'all'.

  testing
make[6]: Nothing to be done for 'all'.

xdp-filter
    GEN      xdp-filter.8
Debugger entered--Lisp error: (file-missing "Searching for program" "No such file or directory" "/usr/bin/env bash")
  call-process("/usr/bin/env bash" nil t nil "-c" "git log -1 --pretty='format:%cI' -- README.org")
  call-process-shell-command("git log -1 --pretty='format:%cI' -- README.org" nil t)
  shell-command("git log -1 --pretty='format:%cI' -- README.org" t)
  shell-command-to-string("git log -1 --pretty='format:%cI' -- README.org")
  (let* ((file-modtime (file-attribute-modification-time (file-attributes filename))) (git-logtime (shell-command-to-string (format "git log -1 --pretty='format:%%cI' -- %s" filename))) (git-modtime (condition-case nil (progn (parse-iso8601-time-string git-logtime)) (error nil)))) (or git-modtime file-modtime))
  get-file-mod-time("README.org")
  (let* ((feat-list (split-string enabled-features)) (org-export-exclude-tags (get-feature-values feat-list feature-exclude-tags)) (modtime (get-file-mod-time infile))) (save-current-buffer (set-buffer (open-file infile)) (org-export-to-file 'man outfile) (filter-post-export outfile feat-list version modtime)))
  export-man-page("xdp-filter.8" "README.org" "LIBBPF_PERF_BUFFER__CONSUME LIBBPF_BTF__LOAD_FROM_..." "v1.2.8")
  eval((export-man-page "xdp-filter.8" "README.org" "LIBBPF_PERF_BUFFER__CONSUME LIBBPF_BTF__LOAD_FROM_..." "v1.2.8") t)
  command-line-1(("--load" "../lib/export-man.el" "--eval" "(export-man-page \"xdp-filter.8\" \"README.org\" \"LIBB..."))
  command-line()
  normal-top-level()

make[5]: *** [../lib/common.mk:126: xdp-filter.8] Error 255
make[4]: *** [Makefile:34: xdp-filter] Error 2
make[4]: Leaving directory '/scratch/local2/ow/build_dir/target-aarch64_cortex-a53_musl/xdp-tools-1.2.8'
make[3]: *** [Makefile:139: /scratch/local2/ow/build_dir/target-aarch64_cortex-a53_musl/xdp-tools-1.2.8/.built] Error 2
make[3]: Leaving directory '/scratch/local2/ow/package/network/utils/xdp-tools'
time: package/network/utils/xdp-tools/compile#1.78#0.37#2.10
    ERROR: package/network/utils/xdp-tools failed to build.
make[2]: *** [package/Makefile:116: package/network/utils/xdp-tools/compile] Error 1
make[2]: Leaving directory '/scratch/local2/ow'
make[1]: *** [package/Makefile:110: /scratch/local2/ow/staging_dir/target-aarch64_cortex-a53_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/scratch/local2/ow'
make: *** [/scratch/local2/ow/include/toplevel.mk:231: world] Error 2
```

--- a/lib/export-man.el
+++ b/lib/export-man.el
@@ -42,7 +42,8 @@
 
 (defun get-file-mod-time (filename)
   (let* ((file-modtime (file-attribute-modification-time (file-attributes filename)))
-         (git-logtime (shell-command-to-string (format "git log -1 --pretty='format:%%cI' -- %s" filename)))
+         (git-logtime (ignore-errors (shell-command-to-string
+                                      (format "git log -1 --pretty='format:%%cI' -- %s" filename))))
          (git-modtime (ignore-errors (parse-iso8601-time-string git-logtime))))
     (or git-modtime file-modtime)))
 

